CRD cleanup

aruniiird opened the pull request:
Cleaned up unwanted data structures (like dashboard, enabled field etc).
Changed the class name from 'StorageAlerts' to 'StorageAlert'

Fix Issue: https://github.com/monstorak/monstorak/issues/26

Signed-off-by: aruniiird <arun.iiird@gmail.com>

aruniiird committed:
CRD cleanup

Cleaned up unwanted data structures (like dashboard, enabled field etc).
Changed the class name from 'StorageAlerts' to 'StorageAlert'

Signed-off-by: aruniiird <arun.iiird@gmail.com>

umangachapagain commented on deploy/crds/alerts_v1alpha1_storagealert_cr.yaml:
```diff
@@ -0,0 +1,7 @@
+metadata:
+  name: example-storagealert
+spec:
+  # Add fields here
```
Can you add the fields here? 
empty spec is error prone.

aruniiird replied:
@umangachapagain , have made the the changes, please take a look...

aruniiird committed:
Filled details to 'alerts_v1alpha1_storagealert_cr.yaml' file

Filled in some default details to CR file.
Renamed the 'StorageName' type to 'StorageType'

Signed-off-by: aruniiird <arun.iiird@gmail.com>

aruniiird committed:
Changing the 'prometheusNamespace' value

Changed 'prometheusNamespace' value in CR to 'openshift-monitoring'

Signed-off-by: aruniiird <arun.iiird@gmail.com>
